extensions/ql-vscode/src/remote-queries/shared/variant-analysis.ts

```diff
@@ -113,6 +114,7 @@ export interface VariantAnalysisSubmission {
     name: string,
     filePath: string,
     language: VariantAnalysisQueryLanguage,
+    queryText: string,
```

Comment:
Why is this repeated here?

Reply:
To construct the `VariantAnalysis`, we initially use the `VariantAnalysisSubmission`. Since the value is not in the returned API response, that means it must be added to the `VariantAnalysisSubmission`